docs/kubernetes/index.rst

```diff
@@ -1,140 +1,162 @@
+| node                         | X                      |                                 | All supported Kubernetes resources                                 |
++------------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+| partition                    |                        | X                               | The |kctlr| cannot create or destroy BIG-IP partitions.            |
++------------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+| pool                         | X                      |                                 | Applies to all supported Kubernetes resources                      |
++------------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+| pool member                  | X                      |                                 | Applies to all supported Kubernetes resources                      |
++------------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+| self IP                      |                        | X                               | Applies to all supported Kubernetes resources                      |
++------------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+| profiles [#prof]_                                                                                                                                            |
++---+--------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+|   | HTTP                     |                        | X                               | :ref:`Kubernetes Ingress Resources <kctlr-ingress-config>`         |
+|   |                          |                        |                                 | (L7) only                                                          |
++---+--------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+|   | TCP                      |                        | X                               | Applies to all supported Kubernetes resources                      |
++---+--------------------------+------------------------+---------------------------------+--------------------------------------------------------------------+
+|   | SSL                      | X                      | X                               | Applies to all supported Kubernetes resources                      |
```

Comment:
Right now ConfigMap will only configure client SSL profiles. Routes can configure server and client and create both. Ingress also can only use client profiles. We should break this out or clarify this a little as there is different functionality currently between the differing methods.